packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -137,11 +136,15 @@ export class MonacoEditorLanguageClientWrapper {
         await this.editorApp?.updateDiffModel(modelUpdate);
     }
 
-    async updateEditorOptions(options: editor.IEditorOptions & editor.IGlobalEditorOptions | VscodeUserConfiguration): Promise<void> {
-        if (this.editorApp) {
-            await this.editorApp.updateEditorOptions(options);
+    updateUserConfiguration(config: UserConfiguration) {
+        return this.editorApp?.updateUserConfiguration(config);
+    }
+
+    async updateEditorOptions(options: editor.IEditorOptions & editor.IGlobalEditorOptions): Promise<void> {
+        if (this.editorApp && this.editorApp.getAppType() === 'classic') {
+            await (this.editorApp as EditorAppClassic).updateMonacoEditorOptions(options);
```

Comment:
This feels like a bit of a smell if this can only be called in the classic configuration. Maybe something to address later on.

Reply:
I will remove this method.